package-lock.json

```diff
@@ -5582,7 +5620,8 @@
         },
         "kind-of": {
           "version": "6.0.2",
-          "resolved": ""
```

Comment:
That's interesting

Reply:
Seems better?

Reply:
ya, but not sure how the original happened